keyboards/yandrstudio/eau_r2/eau_r2.c

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2022 JasonRen(biu)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "eau_r2.h"
+
+void keyboard_post_init_kb(void) {
+#   ifdef RGBLIGHT_ENABLE
+    rgblight_reload_from_eeprom();
+#   endif
+}
+
```

Comment:
```suggestion
```

Reply:
In some STM32 cases, adding this code can avoid the abnormal lighting of the first RGB, and using PWM as the driver can also avoid this problem.